config_hub/pretrain/debug.yaml

```diff
@@ -31,6 +31,7 @@ devices: auto
 seed: 42
 data: OpenWebText
 out_dir: out/pretrain/debug
+checkpoint_dir: null
```

Comment:
What do you think about calling this `tokenizer_dir` or similar? A slight problem right now is that this could be confused with "--resume", but if you passed a checkpoint dir to resume to `--checkpoint_dir` it would silently not do what you meant.

Reply:
Fine with me. Do you want to make the change or should I?